source/manual/incident-fixing.html.md

```diff
@@ -0,0 +1,21 @@
+---
+owner_slack: "@ana.botto"
+title: Fix an Incident
+parent: "/manual.html"
+layout: manual_layout
+type: learn
+section: Incidents
+---
+
+When there is an incident in an application you are supporting, it is important to make a quick fix so that users can get back to use the application as usual.
+
+You should look into the logs on Sentry and Logit to see where the error is coming from.
```

Comment:
I wouldn't say 'should' in this manner, as it isn't a mandatory requirement - we're offering a list of suggestions of where someone _might_ look

Reply:
Worth trying to respect the sort of semantic requirements that are used for RFCs, as they're clear and well documented:

https://datatracker.ietf.org/doc/html/rfc2119